### DISCUSSION TITLE
How to open Google login in a new/separated window?

### DISCUSSION THREAD

**Discussion by jonalxh** [nuxt-community/auth-module#1456] [Q&A]
Hi, I want to know if it is possible to open the google auth page in a new window (maybe like a popup), because I don't want user to be redirected in the same page.

Thanks.

**Comment by lolbyaka** [comment]
Same issue. Any updates?

**Comment by hamzamogni** [comment]
any updates?

**Comment by bsor-dev** [comment]
+1 how to make loginWith in new window/popup. Its doable using google js library but not sure how in `$auth`
